### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GitHubEmploy**
Adding instruction to download the installer file for the linux distro, in case it was not obvious before

Requested here:
https://discord.com/channels/868013470023548938/868013470518505494/1309394857181315142
https://discord.com/channels/868013470023548938/868013470518505494/1309395939320135710

### CODE REVIEW

**Comment by GitHubEmploy** [comment]
> This still isn't 100% clear to a newbie that they need to also install that. Just adding something like "sudo apt install betaflight-configurator_10.10.0_amd64.deb" after the download would be nice. Unless that should not be done until after the other dependencies.
> 
> I think
> 
> ```
> wget https://github.com/betaflight/betaflight-configurator/releases/download/10.10.0/betaflight-configurator_10.10.0_amd64.deb
> sudo apt install betaflight-configurator_10.10.0_amd64.deb
> ```

Hello,

Not adding that step right after was extremely intentional, as the install will fail on ubuntu 24.10 if you do these directly as shown in the screenshot below:
image

The intent is that they would first download that file, and the continue with the steps needed for their distro.

**Comment by thespova** [comment]
Ah got it.

**Comment by thespova** [comment]
Maybe both those steps should be added after the dependencies than? Or maybe its fine the way it is. IDK thanks.

**Comment by nerdCopter** [comment]
This is no longer relevant as there will be no further binary releases.
From 11.0, there will only be https://app.betaflight.com/, https://master.dev.app.betaflight.com/, and maybe version specific URL's.

**Comment by nerdCopter** [comment]
i'll redact my comment/closing since the how-to was recently added.   but really this makes no sense, since this readme is attached to current master at 11.0, not legacy 10.10.

File: README.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ Changes to the security model used in the latest versions of MacOS X 10.14 (Moja
 
 #### Linux users
 
+First step is to download the installer and keep it in your active directory, which can be done with the following command:
```
Comment (haslinghuis) [inline README.md:42]: ADDS :)
```suggestion
First step is to download the installer and keep it in your working directory, which can be done with the following command:
```

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**